plugins/DuskVerb/tools/tuner/full_check.py

```
@@ -0,0 +1,855 @@
+        passing = abs(delta) <= gate
+        return delta, ('PASS' if passing else 'FAIL'), \
+               f"  {label:30s}  DV={dv_val:.3f}  Lex={lex_val:.3f}  Δ={delta:+6.1f}%  gate=±{gate}%  {'✓' if passing else '✗'}"
+    else:
+        delta = dv_val - lex_val
+        passing = abs(delta) <= gate
+        return delta, ('PASS' if passing else 'FAIL'), \
+               f"  {label:30s}  DV={dv_val:+7.3f}  Lex={lex_val:+7.3f}  Δ={delta:+6.2f}  gate=±{gate}  {'✓' if passing else '✗'}"
+
+
+def audit(dv_dir, lex_dir, name='preset', category=''):
+    dv_dir = Path(dv_dir); lex_dir = Path(lex_dir)
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)